cmd/istioctl_test.go

```diff
@@ -60,10 +60,7 @@ func TestIstioctl_istioctlArgChecks(t *testing.T) {
 	_, err = f.Write(raw)
 	require.NoError(t, err)
 
-	require.NoError(t, os.Setenv("GETMESH_TEST_MANIFEST_PATH", f.Name()))
-	defer func() {
-		require.NoError(t, os.Setenv("GETMESH_TEST_MANIFEST_PATH", ""))
-	}()
+	t.Setenv("GETMESH_TEST_MANIFEST_PATH", f.Name())
```

Comment:
good find, TIL, https://github.com/golang/go/pull/41857 says this will disable running tests in parallel
@mathetake this doesn't need to run in parallel right ? I only see e2e tests running with `t.Parallel()`

Reply:
yep!

Reply:
@arkodg @mathetake
Do we have the reason why we don't run unit tests in parallel using `t.Parallel()` ? I meant test cases in the `internal` package.